src/constants.ts

```diff
@@ -11,6 +11,14 @@ export const RPC_HTTP_PORT = 9944;
 // The port substrate listens for p2p connections on
 export const P2P_PORT = 30333;
 
+// Paths for downloading the binaries for zombienet setup
+export const DEFAULT_COMMAND_URL =
+  "https://github.com/paritytech/polkadot/releases/download/v0.9.27/polkadot";
+export const DEFAULT_ADDER_COLLATOR_URL =
+  "https://gitlab.parity.io/parity/mirrors/polkadot/-/jobs/1769497/artifacts/raw/artifacts/adder-collator";
+export const DEFAULT_CUMULUS_COLLATOR_URL =
+  "https://github.com/paritytech/cumulus/releases/download/v0.9.230/polkadot-parachain";
```

Comment:
same here.

Reply:
For Parachain URL we cannot really do that as the latest release may not contain the `polkadot-parachain` asset (see [the latest one](https://github.com/paritytech/cumulus/releases/tag/parachains-v9270) for example. I will make a change for the polkadot, but we will need probably, until somehow this is standardized, to adjust the `polkadot-parachain` manually for now